### PR TITLE
feat(schema): improve json schemas

### PR DESCRIPTION
**Pull request by hekike**
Add metadata to JSON schemas.

### CODE REVIEW

File: internal/ingest/kafkaingest/schema/event_key.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "type": "string"
-}
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://openmeter.io/event_key.schema.json",
+    "type": "string",
+    "title": "Event Key",
+    "description": "Event Key Schema"
+}
```
Comment (hekike) [inline internal/ingest/kafkaingest/schema/event_key.json:7]: weird, autoformat removes new line at end

File: internal/ingest/kafkaingest/schema/event_key.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "type": "string"
-}
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://openmeter.io/event_key.schema.json",
```
Comment (tothandras) [inline internal/ingest/kafkaingest/schema/event_key.json:3]: Do we need to serve these on their url?

Comment (hekike) [reply internal/ingest/kafkaingest/schema/event_key.json:3]: Good questions. Maybe we should set them to their relative path in the repo, see:
https://json-schema.org/understanding-json-schema/structuring.html#id

Comment (hekike) [reply internal/ingest/kafkaingest/schema/event_key.json:3]: Also, maybe it's fine:

> Even though schemas are identified by URIs, those identifiers are not necessarily network-addressable. They are just identifiers. Generally, implementations don’t make HTTP requests (https://) or read from the file system (file://) to fetch schemas. Instead, they provide a way to load schemas into an internal schema database. When a schema is referenced by it’s URI identifier, the schema is retrieved from the internal schema database.

**Review by tothandras — APPROVED** [review]

**Pull request merged**